packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2390,6 +2390,22 @@ const schema: SchemaType<DbPost> = {
     canRead: ['guests'],
   },
   
+  topLevelCommentCount: {
+    type: Number,
+    optional: true,
+    defaultValue: 0,
+    
+    ...denormalizedCountOfReferences({
```

Comment:
Trade-off vs. making this a resolver field?  Feels like we're constantly running into data consistency issues with `denormalizedCountOfReferences` fields